apps/web/src/components/ui/healthcare/healthcare-loading.tsx

```
@@ -0,0 +1,566 @@
+
+    // Error icon based on type
+    const ErrorIcon = React.useMemo(() => {
+      switch (type) {
+        case 'network':
+          return (
+            <svg
+              className='h-5 w-5 text-orange-600'
+              fill='none'
+              viewBox='0 0 24 24'
+              stroke='currentColor'
+            >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)